Update README.md

selvats opened the pull request:
Updated this page to indicate that it is for the Lacework Linux agent. Also suggested some content changes and updated the URLs.

selvats committed:
Update README.md

Updated this page to indicate that it is for the Lacework Linux agent. Also suggested some content changes and updated the URLs.

anilnanduri commented on README.md:
```diff
@@ -1,37 +1,37 @@
-This README.md file provides installation instructions for installing the agent from this Lacework Agent GitHub repository.
-You can also install the agent from the Lacework Console. For a listing of the different agent install methods, see [Agent Install Options](https://support.lacework.com/hc/en-us/sections/360002375733).
+This README.md file provides instructions for installing the agent from this GitHub repository.
+You can also install the agent from the Lacework Console. For information about the different installation methods, see [Linux Agent Installation Methods](https://docs.lacework.com/onboarding/agent-installation-prerequisites).
```
do you think we should mention "linux agent" here too? (OR) may be put a note on the top saying agent in this document means "linux agent".

May be it is implicit here, we do not need to mentioned linux agent?

anilnanduri commented on README.md:
```diff
@@ -45,7 +45,7 @@ Complete the following prerequisites steps before starting any of the agent inst
        If the verification is successful, an `OK` is reported.
 4. Verify that the checksum is signed correctly.
-    1. Download the Lacework Agent GPG key.
+    1. Download the Lacework agent GPG key.
```
Same here too? Should we mention linux?

anilnanduri commented on README.md:
````diff
@@ -132,13 +132,13 @@ For single host installations, Lacework recommends using the installation script
-4. Install the agent on Kubernetes by following the instructions in [Deploy on Kubernetes](https://support.lacework.com/hc/en-us/articles/360005263034).
-   On the **Releases** page of this site, find the appropriate ```docker pull``` command for a specific release under the **Lacework Agent Docker image** heading.
+4. Install the agent on Kubernetes by following the instructions in [Deploy on Kubernetes](https://docs.lacework.com/onboarding/deploy-on-kubernetes).
+   On the **Releases** page of this site, find the appropriate ```docker pull``` command in the **Lacework Agent Docker Images** section for a specific release.
````
Same here too? should we say linux agent?

anilnanduri commented on README.md:
````diff
@@ -151,12 +151,12 @@ For single host installations, Lacework recommends using the installation script
-4. Install the agent by following the instructions in [Install using Docker Swarm](https://support.lacework.com/hc/en-us/articles/360005321473).
-   On the **Releases** page of this site, find the appropriate `docker pull` command for a specific release under the **Lacework Agent Docker image** heading.
+4. Install the agent by following the instructions in [Install using Docker Swarm](https://docs.lacework.com/onboarding/install-using-docker-compose).
+   On the **Releases** page of this site, find the appropriate ```docker pull``` command in the **Lacework Agent Docker Images** section for a specific release.
````
same here too, linux agent?

selvats committed:
Update README.md

Incorporated Anil's feedback by updating section titles in the document to indicate that the content in the sections apply to the Linux agent.
